doc/contributing.rst

```diff
@@ -33,33 +33,39 @@ Quickstart Guide
 
 1. Fork the project on GitHub (using the fork button) and create a local clone, if you haven’t already:
 
-  ``git clone https://github.com/user_name/MITgcm66h.git``
+::
+
+    % git clone https://github.com/user_name/MITgcm.git
 
-2. Move into your local clone directory (cd MITgcm66h) and and set up a remote that points to the original:
+2. Move into your local clone directory (cd MITgcm) and and set up a remote that points to the original:
 
-  ``git remote add upstream https://github.com/altMITgcm/MITgcm66h.git``
+::
+
+    % git remote add upstream https://github.com/altMITgcm/MITgcm.git
 
 3. This step is optional, but highly recommended. It will ensure that your GitHub repo stays up to date with the main repository. Checkout the master branch and sync everywhere:
 
-|  ``git checkout master``
-|  ``git pull upstream master``
-|  ``git push origin master``
-|  
+::
 
+   % git checkout master
+   % git pull upstream master
+   % git push origin master
```

Comment:
A couple things on the github integration:
  - There is a community document that explains workflows that is meant to be linked into sphinx documents:  https://github.com/matthew-brett/gitwash  You may want to consider just including that instead of having your own set of instructions.  Not that the instructions here are bad, but `gitwash` was made to stop every group from typing these out each time...
  - That document suggests deleting your master branch on your local fork. http://matplotlib.org/devel/gitwash/development_workflow.html#consider-deleting-your-master-branch   I've found that super useful because it stops me from trying to add pull requests from my local master.

Reply:
Thanks @jklymak - we'll have a look at these.